### ISSUE TITLE
PK SIM COMPARISON

### ISSUE THREAD

**Issue by audreyoc** [Open-Systems-Pharmacology/PK-Sim#2686]
How do I make a population simulation comparison? Whenever I create the population simulation comparison, the simulation data never goes into the comparison. There is no output and no PK curve.

**Comment by audreyoc** [comment]
![image](https://github.com/Open-Systems-Pharmacology/PK-Sim/assets/138920486/c6ac940d-77ea-4004-aed6-fb893b6c6cd2)

**Comment by audreyoc** [comment]
![image](https://github.com/Open-Systems-Pharmacology/PK-Sim/assets/138920486/9ec58f4d-fb8b-4d03-a54b-3fc95c0ce1b0)

**Comment by Yuri05** [comment]
Did you calculate the simulation outputs in the population simulations which you are trying to compare?
If you have more than 1 output in simulations: did you assign it in the comparison?

![grafik](https://github.com/Open-Systems-Pharmacology/PK-Sim/assets/25061876/0d1383d7-70d5-43e3-a020-b98c67e3728e)

![grafik](https://github.com/Open-Systems-Pharmacology/PK-Sim/assets/25061876/6902e745-0b3b-4fda-90d4-bf31a378ecb1)

![grafik](https://github.com/Open-Systems-Pharmacology/PK-Sim/assets/25061876/5deeda3d-ed32-4c6b-b7ea-6e3203d83b4d)

**Comment by audreyoc** [comment]
> Did you calculate the simulation outputs in the population simulations which you are trying to compare? If you have more than 1 output in simulations: did you assign it in the comparison?
> 
> ![grafik](https://user-images.githubusercontent.com/25061876/254930495-0d1383d7-70d5-43e3-a020-b98c67e3728e.png)
> 
> ![grafik](https://user-images.githubusercontent.com/25061876/254929069-6902e745-0b3b-4fda-90d4-bf31a378ecb1.png)
> 
> ![grafik](https://user-images.githubusercontent.com/25061876/254929910-5deeda3d-ed32-4c6b-b7ea-6e3203d83b4d.png)

Thank you for your response. My output is plasma concentration. When I create the simulation, there are no outputs to choose from.

**Comment by audreyoc** [comment]
@Yuri05 Is it because I'm using two different drugs?

**Comment by Yuri05** [comment]
>Is it because I'm using two different drugs?

Yes, this might be a reason. Comparison will only allow choosing an output which is the same between simulations.
"Same" means here: same location, same drug.
